### PR TITLE
change column names to match original schema

### PR DESCRIPTION
**Pull request by AlaaSarhan**
https://phabricator.wikimedia.org/T220157

### CODE REVIEW

File: src/PackagePrivate/Doctrine/DoctrinePropertyTermStore.php [hunk 1]
```diff
@@ -193,11 +193,11 @@ public function getTerms( PropertyId $propertyId ): Fingerprint {
...
-SELECT text, language, type_id FROM {$this->tableNames->propertyTerms()}
-INNER JOIN {$this->tableNames->termInLanguage()} ON {$this->tableNames->propertyTerms()}.term_in_lang_id = {$this->tableNames->termInLanguage()}.id
-INNER JOIN {$this->tableNames->textInLanguage()} ON {$this->tableNames->termInLanguage()}.text_in_lang_id = {$this->tableNames->textInLanguage()}.id
-INNER JOIN {$this->tableNames->text()} ON {$this->tableNames->textInLanguage()}.text_id = {$this->tableNames->text()}.id
-WHERE property_id = ?
+SELECT wbx_text, wbxl_language, wbtl_type_id FROM {$this->tableNames->propertyTerms()}
+INNER JOIN {$this->tableNames->termInLanguage()} ON {$this->tableNames->propertyTerms()}.wbpt_term_in_lang_id = {$this->tableNames->termInLanguage()}.wbtl_id
+INNER JOIN {$this->tableNames->textInLanguage()} ON {$this->tableNames->termInLanguage()}.wbtl_text_in_lang_id = {$this->tableNames->textInLanguage()}.wbxl_id
+INNER JOIN {$this->tableNames->text()} ON {$this->tableNames->textInLanguage()}.wbxl_text_id = {$this->tableNames->text()}.wbx_id
+WHERE wbpt_property_id = ?
```
Comment (JeroenDeDauw) [inline src/PackagePrivate/Doctrine/DoctrinePropertyTermStore.php:200]: If we have the prefixes we can drop the table names. Think that is a good idea here?

**Review by JeroenDeDauw — APPROVED** [review]
I think this makes things worse from a maintainability PoV so am not keen on it. That's several levels of importance below the architecture concerns so I'm ok with doing this if we really need to.

**Review by JeroenDeDauw — APPROVED** [review]

**Comment by JeroenDeDauw** [comment]
Haha, you can now approve something multiple times :D

File: src/PackagePrivate/Doctrine/DoctrinePropertyTermStore.php (hunk 1 above)
Comment (AlaaSarhan) [reply src/PackagePrivate/Doctrine/DoctrinePropertyTermStore.php:200]: do you mean we would have only `wbpt_id` ?

**Comment by AlaaSarhan** [comment]
> I think this makes things worse from a maintainability PoV so am not keen on it. That's several levels of importance below the architecture concerns so I'm ok with doing this if we really need to.

Yeap don't like myself either .. very weird convention imo but we have to follow it as long as we are putting things in same DB maybe

Actually @Ladsgroup is renaming a column name a costly operation on DB level .. just to know if we can do it later so that we keep it in mind

File: src/PackagePrivate/Doctrine/DoctrinePropertyTermStore.php (hunk 1 above)
Comment (JeroenDeDauw) [reply src/PackagePrivate/Doctrine/DoctrinePropertyTermStore.php:200]: Yes

Comment (AlaaSarhan) [reply src/PackagePrivate/Doctrine/DoctrinePropertyTermStore.php:200]: won't work :sweat: cause we have already that column as the record id

**Comment by JeroenDeDauw** [comment]
AFAIK the convention is there to make querying things easier, in particular when using the MW DBAL. If we consider the tables to be private to this component, that simply does not matter.

**Pull request merged**